test/get_weets.spec.js

```diff
@@ -0,0 +1,35 @@
+const supertest = require('supertest');
+
+const app = require('../app');
+const { AUTHORIZATION_ERROR } = require('../app/errors');
+const { mockUser } = require('./mocks/mockUsers.js');
+
+const request = supertest(app);
+const createUser = user => request.post('/users').send(user);
+const signIn = user => request.post('/users/sessions').send(user);
+const getWeets = (token, page = 1, limit = 10) =>
+  request
+    .get('/weets')
+    .set('Authorization', `Bearer ${token}`)
+    .query({ page, limit });
+
+let userToken = null;
+beforeAll(async () => {
+  await createUser(mockUser);
+  const signInData = await signIn({ email: mockUser.email, password: mockUser.password });
+  userToken = signInData.body.token;
+});
+
+describe('GET /users/sessions', () => {
```

Comment:
If the following that I wrote is in another card, discard the following: I would add more tests on pagination

Reply:
i implemented factory and test for pagination